packages/agents/src/mcp/index.ts

```diff
@@ -738,8 +737,8 @@ export abstract class McpAgent<
           const id = namespace.idFromName(`sse:${sessionId}`);
           const doStub = namespace.get(id);
 
-          // Forward the request to the Durable Object
-          const error = await doStub.onSSEMcpMessage(sessionId, request);
+          const messageBody = await request.json();
+          const error = await doStub.onSSEMcpMessage(sessionId, messageBody);
```

Comment:
I was struggling to get the tests to pass when passing the entire request to `onSSEMcpMessage`. I noticed that the only thing it actually needs is the message body so being explicit seemed okay.